### ISSUE TITLE
[PS 1.7.8] Uncaught Error: Class 'AppKernel' not found

### ISSUE THREAD

**Issue by takestop** [PrestaShop/PrestaShop#26484]
hi everyone, we used a private cloud with PHP 7.4 on which prestashop version 1.7.8 was installed. We installed the MigrationPro module on the new version of prestashop which, by connecting to the old private cloud, migrated all the data relating to the database. Once the migration is complete, we have configured the new prestashop 1.7.8 with the various data (couriers, suppliers, brands, etc.). I installed the marketplace modules (amazon, eprice, ibs) for the price / quantity synchronization and at the end I proceeded to request the modification of the cron to our hosting by updating with the new ones. Until this morning at 12.30 everything was working perfectly but when I tried again around 13.00 the site was blown without any changes to any part of prestashop. The errors are these below:

```
Warning: include(/home/web/www.takestop.com/www/vendor/composer/../../app/AppKernel.php): failed to open stream: No such file or directory in /home/web/www.takestop.com/www/vendor/composer/ClassLoader.php on line 480

Warning: include(): Failed opening '/home/web/www.takestop.com/www/vendor/composer/../../app/AppKernel.php' for inclusion (include_path='/home/web/www.takestop.com/www/vendor/pear/pear_exception:/home/web/www.takestop.com/www/vendor/pear/pear-core-minimal/src:/home/web/www.takestop.com/www/vendor/pear/archive_tar:/home/web/www.takestop.com/www/vendor/pear/console_getopt:.:/usr/share/php') in /home/web/www.takestop.com/www/vendor/composer/ClassLoader.php on line 480

Fatal error: Uncaught Error: Class 'AppKernel' not found in /home/web/www.takestop.com/www/config/autoload.php:28 Stack trace: #0 /home/web/www.takestop.com/www/config/config.inc.php(38): require_once() #1 /home/web/www.takestop.com/www/admin373gqi05e/index.php(40): require('/home/web/www.t...') #2 {main} thrown in /home/web/www.takestop.com/www/config/autoload.php on line 28
```
https://www.takestop.com

What could have happened? we can't trace the problem and not even our hosting staff can.

**Comment by hibatallahAouadni** [comment]
Hello @takestop 

Thanks for your issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template to submit a complete bug report**.

Read more about how we expect the issues to be handled [here](https://build.prestashop.com/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it.

Thanks!
